### ISSUE TITLE
RFE: TAXII Channels

### ISSUE THREAD

**Issue by JasonKeirstead** [oasis-tcs/cti-taxii2#3]
TAXII requires a data-bus type of mechanism where data can be posted to subscribers for receipt without the implementor of the TAXII server having to worry about storing that data for all time, as they currently have to with TAXII collections.

It has been suggested that this use case might also be able to be fulfilled by adding some notion of TTL to TAXII collections. I will leave this open to the possibility that this may also be a way to fill the use case.

**Comment by jordan2175** [comment]
This is related to issue #2

**Comment by MarkDavidson** [comment]
Recommendation: Discuss scope and goals at F2F

**Comment by gtback** [comment]
Is there any requirement that items in collections be kept forever? [Section 3.4.2](http://docs.oasis-open.org/cti/taxii/v2.0/cs01/taxii-v2.0-cs01.html#_Toc496542717) of the spec suggests that this is not the case [emphasis added]:

> This can happen if items have been added **or deleted** between subsequent requests.


That doesn't negate the fact that you may want to communicate this type of information to clients, of course.

**Comment by JasonKeirstead** [comment]
@gtback No, of course not. The items that are present in a collection are totally up to the implementation.
